nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/entity/ProcessGroupRecursivity.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.nifi.web.api.entity;
 
-public enum ProcessGroupUpdateStrategy {
-    CURRENT_GROUP,
-    CURRENT_GROUP_WITH_CHILDREN
+public enum ProcessGroupRecursivity {
```

Comment:
This name change impacts the REST API parameters, which also impacts UI JavaScript code. It would be better to separate these changes into a distinct PR specific to the main branch so that the resolution for the integration test issue could be backported to the support branch.